Description: Comment out debugging display routines.
 Otherwise, we get a FTBFS since Debian factory is compiled with
 --disable-streamio.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-06-04

--- a/M2/Macaulay2/e/x-factor.cpp
+++ b/M2/Macaulay2/e/x-factor.cpp
@@ -229,7 +229,7 @@
 
     // debugging display routines to be called from gdb
     // needs factory to be configured without option --disable-streamio
-#if 1
+#if 0
 void showvar(Variable &t) { std::cout << t << std::endl; }
 void showcf(CanonicalForm &t) { std::cout << t << std::endl; }
 void showcfl(CFList &t) { std::cout << t << std::endl; }
